Sources/Rendering/OpenGL/Convolution2dPass/index.js

```diff
@@ -10,9 +10,9 @@ import { Representation } from 'vtk.js/Sources/Rendering/Core/Property/Constants
 const { vtkErrorMacro } = macro;
 // ----------------------------------------------------------------------------
 
-function vtkCustomPostProcessingPass(publicAPI, model) {
+function vtkConvolution2dPass(publicAPI, model) {
   // Set our className
-  model.classHierarchy.push('vtkCustomPostProcessingPass');
+  model.classHierarchy.push('vtkConvolution2dPass');
```

Comment:
A capital D is more consistent here (`vtkConvolution2DPass`)

Reply:
Will be changed.